README.md

```diff
@@ -219,7 +219,29 @@ python config/fypp_deployment.py --help
 git checkout stdlib-fpm
 fpm build --profile release
 ```
-#### Runing the examples
+
+
+#### Installing with fpm
+
+Either option you chose for building the `stdlib`, you can install it with:
+```sh
+fpm install --profile release
+```
+The command above will install the following files:
+- `libstdlib.a` into `~/.local/lib/`
```

Comment:
Maybe add : into `~/.local/lib/` (Unix) or `C:\Users\<username>\AppData\Roaming\local` (Windows)

Reply:
Added a commit below, I was wondering whether also the next line needs to be modified for Windows (although I don't know where windows would store the `.[s]mod` files)

Reply:
Same pattern, what changes is that longer root, so it would be:
`C:\Users\<username>\AppData\Roaming\local\lib` and `C:\Users\<username>\AppData\Roaming\local\include`